plaidml2/op/op_test.cc

```diff
@@ -155,10 +229,9 @@ TEST(Op, Sum) {
 
 TEST(Op, Squeeze) {
   auto A = Placeholder(PLAIDML_DATA_FLOAT32, {32, 1, 4, 1}, "A");
-  auto t = op::squeeze( //
-      A,             // tensor to squeeze
-      {1, 3}         // axes to squeeze
-  );
+  auto t = op::squeeze(  //
+      A,                 // tensor to squeeze
+      {1, 3} /* axes to squeeze*/);
```

Comment:
The formatting of these was fixed in master, could you resolve these in your branch?

Reply:
Done :)